modules/apps/object/object-dynamic-data-mapping-form-field-type/src/main/java/com/liferay/object/dynamic/data/mapping/form/field/type/internal/object/relationship/ObjectRelationshipDDMFormFieldTemplateContextContributor.java

```diff
@@ -235,6 +235,10 @@ private String _getValueKey(DDMFormField ddmFormField) {
 		return systemObjectDefinitionMetadata.getRESTDTOIdPropertyName();
 	}
 
+	private String _replaceDateLabelKey(String date) {
```

Comment:
About the parameter name. In this kind of situations, we usually link the class name and method name to build a most meaningful variable name.

i. e:  **ObjectField**.get**Name**() = **objectFieldName**